src/models/ballad-in-goblets.js

```diff
@@ -42,16 +42,29 @@ export default class BalladInGoblets extends BaseGacha {
     return this.probabilityRange[this.generateRandomNumber(this.probabilityRange.length)]
   }
   getRandomItem(rating) {
-    const itemsList = this.getDrops(rating)
-    const item = itemsList[this.generateRandomNumber(itemsList.length)]
+    const itemsList = this.getDrops(rating);
+    let item;
+
+    // If our previous SSR didn't drop a Venti, then this time, we'll get him.
+    if (this.guaranteedVenti && rating === 5) {
+      item = this.drops.find((el) => el.name === 'Venti');
```

Comment:
use the `grabAVenti` method here, and for the sake of consitency drop the paranthesis when using a single parameter in arrow functions.